CODE_OF_CONDUCT.md

```diff
@@ -55,7 +55,7 @@ further defined and clarified by project maintainers.
 ## Enforcement
 
 Instances of abusive, harassing, or otherwise unacceptable behavior may be
-reported by contacting the project team at support@nexmo.com. All
+reported by contacting the project team at support@vonage.com. All
```

Comment:
This e-mail needs reverted. `support@vonage.com` isn't used for the API products right now.